frontend/src/components/StyledBtn.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+
+export const StyledButton = styled.button`
+  width: 150px;
+  font-size: 16px;
+  color: #6A7885;
+  height: 40px;
+  margin: 10px;
+  background-color: #CAEBF2;
+  border: 1px solid #9099A5;
+  border-radius: 2px;
+  cursor: pointer;
+  box-shadow: 7px 7px white, 7px 7px 0px 1px #9099A5;
+  &:hover {
+    background-color: rgb(203,200,254);
+  }
+`
```

Comment:
In the Sign Up or Log In form, perhaps you could consider adding some margin to the bottom of the button or padding to the bottom of the form. When an error message is displayed, it's getting a little cramped ➡️ ⬅️